<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QWidget" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>439</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_font">
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_picture">
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_FaceExpression">
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>Send File</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_sendfile">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>23</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="text">
         <string>File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_sendir">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>23</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="text">
         <string>Dir</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cal">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>23</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="text">
         <string>Cal</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_ChatLog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>480</width>
     <height>250</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>250</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>65535</width>
     <height>250</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="close_pushbutton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>0</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>Ã—</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_headimage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>450</y>
     <width>168</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>SEND</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>CLOSE</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
